### PR TITLE
Bug 2126604: change name for Thales provider

### PR DESCRIPTION
**Pull request by SanjalKatiyar**

### CODE REVIEW

File: packages/odf/components/kms-config/kms-config.tsx [hunk 1]
```diff
@@ -29,7 +29,7 @@ const KMSProviders: KMSProvidersType = [
     allowedPlatforms: ['IBMCloud'],
   },
   {
-    name: 'Thales (using KMIP)',
+    name: 'Thales CipherTrust Manager (using KMIP)',
```
Comment (GowthamShanmugam) [inline packages/odf/components/kms-config/kms-config.tsx:32]: Is this displayed on the UI? dont we need translation?

Comment (GowthamShanmugam) [inline packages/odf/components/kms-config/kms-config.tsx:32]: Is this displayed on the UI? dont we need translation?

Comment (SanjalKatiyar) [reply packages/odf/components/kms-config/kms-config.tsx:32]: it's product name, it is better not to translate IMO.

Comment (GowthamShanmugam) [reply packages/odf/components/kms-config/kms-config.tsx:32]: ack

**Review by GowthamShanmugam — APPROVED** [review]

**Pull request merged**